libdevcore/CommonIO.cpp

```diff
@@ -26,6 +26,43 @@
 using namespace std;
 using namespace dev;
 
+std::list<std::list<std::string>> dev::memDumpToList(bytes const& _b, unsigned _w)
```

Comment:
Using linked list for small objects is usually much more expensive than vector due to lots of allocations. Especially if you only push_back(). Also why not use QString here to avoid extra conversion later